src/io/config_auto.cpp

```diff
@@ -476,6 +494,10 @@ void Config::GetMembersFromString(const std::unordered_map<std::string, std::str
 
   GetBool(params, "use_missing", &use_missing);
 
+  GetBool(params, "monotone_precise_mode", &monotone_precise_mode);
+  CHECK(!monotone_precise_mode || !use_missing);
+  CHECK(!monotone_precise_mode || constraints_exist);
+
```

Comment:
I think this file is auto generated, you cannot edit it by hand.
You could have some post-processing in `config.cpp`.

Reply:
I am not sure I understand, I locally tested if these checks worked and they did. What do you mean by the file being auto generated? I am not very familiar with how the library is built I am sorry.